### PR TITLE
GitHub Actions CI

### PR DESCRIPTION
**Pull request by woky**
Add GitHub Actions CI
  
Not everything from Drone is ported yet. Uploading artifacts is missing.

### CODE REVIEW

File: integration-tests/test/rnode.py [hunk 1]
```diff
@@ -414,25 +414,6 @@ def make_node(
     return node
 
 
-def get_absolute_path_for_mounting(relative_path: str, mount_dir: Optional[str]=None)-> str:
```
Comment (nzpr) [inline integration-tests/test/rnode.py:417]: will this break drone once merged?

**Review by nzpr — COMMENTED** [review]
This PR should be merged after it is complete and ready to replace drone, right?

**Comment by woky** [comment]
> This PR should be merged after it is complete and ready to replace drone, right?

Well, technically it could run alongside Drone, but I've identified few issues since I opened the PR. I'll open another one.

File: integration-tests/test/rnode.py (hunk 1 above)
Comment (woky) [reply integration-tests/test/rnode.py:417]: Nope. It's just a confusing function and mostly unused function.

**Pull request closed without merging**